status/status.go

```diff
@@ -11,8 +11,9 @@ const (
 )
 
 type StatusInfo struct {
-	Status   map[string]string `json:"status"`
-	Products []ProductInfo     `json:"products"`
+	Status          map[string]string `json:"status"`
+	Products        []ProductInfo     `json:"products"`
+	WarningMessages []string          `json:"warning_messages"`
```

Comment:
Please use camel-case in json: `warningMessages`.
All json has been changed now to conform to this rule (see https://issues.jboss.org/browse/KIALI-768 )

Reply:
Ok, I didn't know that this apply to Json too